public/css/app.css

```diff
@@ -16428,9 +16428,10 @@ table.sortable > thead th div.mozilla {
 }
 
 .alert-success {
+  color: #222;
```

Comment:
These changes need to be somewhere in the sass files (perhaps resources/assets/sass/_alert.scss?) - public/css/app.css is built from those files and overwritten each time they're built.

Reply:
Pushed.  Have a look now?